app/src/services/version.js

```diff
@@ -93,14 +60,12 @@ const service = {
         .where('versionId', versionId)
         .throwIfNotFound();
 
-      // if other versions exist, make next most recent version current
+      // get number of other remaining versions
       const versionCount = await Version.query(trx)
         .count('id as CNT')
         .where({ objectId: objId });
       const count = parseInt(versionCount[0].CNT);
```

Comment:
Since we only care about the number of remaining rows that match a criteria, why aren't we just using the `resultSize` method instead?
Ref: https://vincit.github.io/objection.js/api/query-builder/other-methods.html#resultsize
```js
const count = await Version.query(trx).resultSize();
```
On a second note, do we even need to get a count for this? We're not using this value anywhere afterwards.